Description: [UBUNTU-ONLY] Remove framework-bundle.
 framework-bundle is being removed from twig-extra-bundle
 because php-symfony-framework-bundle pulls in php-doctrine-bundle
 which causes the whole uninstallability mess.
 So temporary remove this as php-twig-extra-bundle has no
 reverse-{build-}dependencies.
Author: Utkarsh Gupta <utkarsh.gupta@canonical.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2021-06-03

--- a/extra/twig-extra-bundle/composer.json
+++ b/extra/twig-extra-bundle/composer.json
@@ -16,7 +16,6 @@
     ],
     "require": {
         "php": "^7.1.3|^8.0",
-        "symfony/framework-bundle": "^4.3|^5.0",
         "symfony/twig-bundle": "^4.3|^5.0",
         "twig/twig": "^2.4|^3.0"
     },
